File: LeetCode/Container_With_Most_Water.cxx
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    int maxArea(vector<int>& height) {
        int maxArea = INT_MIN;
        for(int i = 0; i < height.size(); i++){
            for(int j = 0; j < height.size(); j++){
                int area = abs(i - j) * min(height[i], height[j]);
                maxArea = max(maxArea, area);
            }
        }
    return maxArea;
    }
};

File: LeetCode/Container_With_Most_Water.java
class Solution {
    public int maxArea(int[] height) {
        int max = Integer.MIN_VALUE;
        for(int i = 0; i < height.length; i++){
            for(int j = 0; j < height.length; j++);
                if(i == j){
                    j++;
                }
                int area = Math.abs(i - j) * 

        }
        return max;
    }
}